web/templates/settings/generate_integration_url_modal.hbs

```diff
@@ -0,0 +1,33 @@
+<div class="new-style">
+    <div class="input-group">
+        <label for="integration-input">{{t "Integration"}}</label>
+        <input type="text" id="integration-input" class="modal_text_input integration-url-parameter" autocomplete="off"/>
+    </div>
+    <div class="input-group">
+        <div class="integration-url-stream-wrapper integration-url-parameter">
+            {{> ../dropdown_widget_with_label
+              widget_name="integration-url-stream"
+              label=(t "Stream")}}
+        </div>
+        <p id="stream-not-specified-notice">{{t "If you do not specify a stream, the bot will send notifications via PMs to the creator of the bot."}}</p>
+    </div>
+    <div class="input-group control-label-disabled">
+        <label class="checkbox">
+            <input type="checkbox" id="integration-url-override-topic" class="integration-url-parameter" disabled/>
+            <span></span>
+        </label>
+        <label class="inline" for="integration-url-override-topic">
+            {{t "Override topic for notifications?"}}
```

Comment:
How about "Send all notifications to a single topic" for this label -- and then we can remove the explanatory paragraph below. (And maybe as a `help_link_widget` with help text "By default, most integrations will send notifications to multiple topics in a thoughtful way.")

Reply:
Hmm, that sounds like a good idea. 

I wonder where should the `help_link_widget` link to. Maybe I can add the following to the first bullet point on [this](https://zulip.com/api/integrations-overview#set-up-an-existing-integration) page:

> After successful setup, notifications will be sent to the specified stream and most integrations will send them to multiple topics in a thoughtful way.

Or do we require a separate page for this? I feel that would be too much for just this one piece of information.